features/roomlist/impl/src/main/res/values-fr/translations.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-  <string name="screen_roomlist_a11y_create_message">"Créer une nouvelle conversation ou un nouveau salon"</string>
+  <string name="screen_roomlist_a11y_create_message">"Créer une nouvelle conversation ou une nouvelle salle"</string>
+  <string name="screen_roomlist_empty_message">"Commencez par envoyer un message à quelqu\'un."</string>
+  <string name="screen_roomlist_empty_title">"Aucune discussion pour le moment."</string>
   <string name="screen_roomlist_main_space_title">"Tous les chats"</string>
```

Comment:
"Tous les chats" may have been improved. In French it means "All the cats". "Toutes les conversations" or just "Conversations" would fit better.